modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import {useNavigate} from 'react-router-dom';
+import {KeyedMutator} from 'swr';
+
+import AssignModal from '../../components/AssignModal';
+import useFormModal from '../../hooks/useFormModal';
+import i18n from '../../i18n';
+import {TestrayTask, UserAccount, testrayTaskImpl} from '../../services/rest';
+
+type TaskHeaderActionsProps = {
+	TestrayTask: TestrayTask;
+	mutateTask: KeyedMutator<any>;
```

Comment:
If you know what mutate is, use the correct interface

```suggestion
	mutateTask: KeyedMutator<TestrayTask>;
```